packages/components/src/Actions/ActionButton/ActionButton.component.js

```diff
@@ -206,7 +206,8 @@ function ActionButton(props) {
 		);
 	}
 
-	if (hideLabel || tooltip || tooltipLabel) {
+	const hasLabel = tooltipLabel || label;
```

Comment:
i don't understand this condition. to me the following proposition is ok but maybe i miss something

![image](https://user-images.githubusercontent.com/26482371/122921775-33403100-d363-11eb-948f-1474bd3c190b.png)

Reply:
if i undestood, the only difference between your needs and the current code is to make `tooltip !== false` prior to the rest